.gitattributes

```diff
@@ -0,0 +1,3 @@
+*               text=auto
```

Comment:
I guess it should be `text=lf` 🤔 ?

Reply:
I think for better compatibility with both Windows and Linux/Mac it should be `auto`, strict line endings are only required for `gradlew*` scripts